### DISCUSSION TITLE
Welcome to catalyst Discussions!

### DISCUSSION THREAD

**Discussion by keithamus** [github/catalyst#134] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by adoublef** [comment]
Hi, I've recently stumbled across this library recently and couldn't find a discord or slack group so seems as though this is the alternative to those.

So far I've been experimenting by attempting to port a Lit project (web audio app)to Catalyst just to see if it's viable for my needs. This seems to be the only web component library that behaves well for non-Node backends (the docs stat something about ruby & I personally use Deno) so really looking forward to getting good with this.

I'm also interested in some of the new things that are planned for v2 such as the forable and loadable abilities (both very useful in my usecase).

Regards

**Reply by keithamus** [reply]
I’d encourage you to continue using Lit, which is a great web component framework and should work excellently with non-Node backends.

**Reply by adoublef** [reply]
Appreciate that recommendation but I think there is an element of cognitive overload for me when wanting to understand how Lit works (as with most JS libraries). I'm very curious of how different teams tackle DOM manipulation with their libraries & I'm a fan of how easy it's been to understand the source code for Catalyst, which is a big draw. v1 seems stable enough for me personally.
